docs/fori_loop.md

```diff
@@ -111,4 +111,4 @@ tensor([11], device='xla:0', dtype=torch.int32))
 tensor([51], device='xla:0', dtype=torch.int32)
 ```
 
-For more example and detailed user guide, please read [this test file](https://github.com/pytorch/xla/blob/master/test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py). PyTorch/XLA would include `while_loop` support in 2.3 for simple test case, complex test case and support for `fori_loop` and `scan` would be added after 2.3
+For more example and detailed user guide, please read [this test file](https://github.com/pytorch/xla/blob/ManfeiBai-patch-87/test/test_while_loop.py). PyTorch/XLA would include `while_loop` support in 2.3 for simple test case, complex test case and support for `fori_loop` and `scan` would be added after 2.3
```

Comment:
consider write an actual example and don't rly on a test

Reply:
please only mention that currently we only should `body_fn` with same `input` and `output` shape

Reply:
> consider write an actual example and don't rly on a test
Thanks, added test example in https://github.com/pytorch/xla/blob/ManfeiBai-patch-87/docs/fori_loop.md#simple-user-guide

> please only mention that currently we only should body_fn with same input and output shape
Thanks, added this point after this sentence